conjure-java-core/build.gradle

```diff
@@ -45,7 +45,9 @@ dependencies {
     compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8'
     compile 'com.google.googlejavaformat:google-java-format'
     compile 'com.google.guava:guava'
-    compile 'com.palantir.conjure:conjure-generator-common'
+    compile 'com.palantir.conjure:conjure-generator-common', {
+        exclude group: 'com.palantir.conjure.java', module: 'conjure-lib'
```

Comment:
why is this necessary?

Reply:
Same reason this is duplicated on line 63, a recent change to conjure-generator-common must have added a new dependency on conjure-lib which breaks gcv write-locks picking between conjure-lib (5.3 from conjure project) and the local project link.

Reply:
Ok, I think we can make gradle's resolution strategy to prefer project dependencies https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ResolutionStrategy.html

Reply:
Ah fantastic, thanks!